.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Preliminary Steps
+
+Please review these preliminary steps before logging your issue. You may find the information referenced may answer or explain the behaviour you are seeing. It'll help us to know you've reviewed this information.
+
+Please confirm you have...
+- [ ] reviewed [How Linguist Works](https://github.com/github/linguist#how-linguist-works),
+- [ ] reviewed the [Troubleshooting](https://github.com/github/linguist#troubleshooting) docs,
+- [ ] considered implementing an [override](https://github.com/github/linguist#overrides),
+- [ ] verified an issue has not already been logged for your issue ([linguist issues](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aissue+repo%3Agithub/linguist)).
+
+## Problem Description
+
+<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->
+
+**Affected repository:** <!-- URL to repository -->
```

Comment:
I'd make these tertiary subheadings, and reword the first one so there's no need for a trailing HTML comment:

~~~markdown
### URL of the affected repository

### Last modified on:
<!-- YYYY-MM-DD -->

... etc
~~~